### DISCUSSION TITLE
Would it be possible to add macros to the Test to run field of Perl5 Test run configuration editor?

### DISCUSSION THREAD

**Discussion by markAtAthena** [Camelcade/Perl5-IDEA#2346] [Ideas]
The Perl5 Test run configuration is somewhat limited in use because it doesn't allow you to use the built in macros to have the currently opened test (.t file) be automatically supplied to the test to run field.

I suspect it is easily added though by doing similar to this 

https://github.com/JetBrains/intellij-community/blob/3b0398ca4e1460bafeab749e3b731d43f642ac6c/platform/execution-impl/src/com/intellij/execution/ui/CommonParameterFragments.java#L50
 
around this area:
 
https://github.com/Camelcade/Perl5-IDEA/blob/master/plugin/core/src/com/perl5/lang/perl/idea/run/prove/PerlTestRunConfigurationEditor.java#L120

If you do not have time to look at this, is that something you would be open to a pull request for?

**Comment by hurricup** [comment]
Worth filling an issue and yes, pull request would be nice. I had a little time to spent to plugin recently, now trying to get back to work, but no promises yet.
